tap_mysql/sync_strategies/common.py

```diff
@@ -144,59 +161,122 @@ def sync_query(cursor, catalog_entry, state, select_sql, columns, stream_version
     LOGGER.info('Running %s', query_string)
     cursor.execute(select_sql, params)
 
-    row = cursor.fetchone()
-    rows_saved = 0
-
     database_name = get_database_name(catalog_entry)
 
+    md_map = metadata.to_map(catalog_entry.metadata)
+    stream_metadata = md_map.get((), {})
```

Comment:
What does this do? 🤔

Reply:
It was pulled from the old lines 166 and 167 (below). Think it grabs the metadata object that came in an earlier message.

Reply:
Ah ok cool.